src/KeyringController.test.ts

```diff
@@ -266,14 +278,17 @@ describe('KeyringController', function () {
       expect(allAccountsAfter[0]).toBe(walletOneAddresses[0]);
     });
 
-    it('throws error if argument password is not a string', async function () {
-      await expect(() =>
-        keyringController.createNewVaultAndRestore(12, walletTwoSeedWords),
-      ).rejects.toThrow('Password must be text.');
+    it('throws error if argument password is not a string', async () => {
+      await expect(async () =>
+        keyringController.createNewVaultAndRestore(
+          12 as any,
```

Comment:
At some point it would be nice to enable a lint rule that prevents the use of `any`. To anticipate this, what do you think about using `@ts-expect-error` to tell TypeScript it's okay that this type doesn't match?